Getting-Started/Code/Creating-Forms/index.md

```diff
@@ -119,7 +123,9 @@ In the partial view, we're going to render our contact form using the view model
 
 
 #### Adding the macro
-The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Right click __Macros__ and select __Reload__ then expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
+The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
+
+Tip: If you don't see your new macro listed, right click __Macros__ and select __Reload__
```

Comment:
Excellent idea putting this into a tip here! 👏 

Have you seen the formatting options we have for tips like these?
You can see them here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips 
Makes the tip stands out a bit more! What do you think?